test/e2e/framework/pv_util.go

```diff
@@ -494,25 +494,22 @@ func testPodSuccessOrFail(c clientset.Interface, ns string, pod *v1.Pod) error {
 // Deletes the passed-in pod and waits for the pod to be terminated. Resilient to the pod
 // not existing.
 func DeletePodWithWait(f *Framework, c clientset.Interface, pod *v1.Pod) error {
+	const maxWait = 15 * time.Minute
```

Comment:
Can we shorten it to something like 5 minutes?  I wouldn't expect volume cleanup to take that long.

Reply:
Sure. Hard to know what the max wait should be. I was erring on the side wanting to wait long enough for the long tail cases.